google/resource_compute_target_pool.go

```diff
@@ -57,6 +58,19 @@ func resourceComputeTargetPool() *schema.Resource {
 				Computed: true,
 				ForceNew: false,
 				Elem:     &schema.Schema{Type: schema.TypeString},
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					// instances are stored in state as "zone/name"
+					oldParts := strings.Split(old, "/")
+
+					// instances can also be specified in the config as a URL
+					r := regexp.MustCompile(fmt.Sprintf(zonalLinkBasePattern, "instances"))
```

Comment:
Ideally we'd compile this to a global variable, not on every DiffSuppressFunc call.

Reply:
Done